SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -428,6 +452,8 @@ bool FLocalDeploymentManager::IsServiceRunningAndInCorrectDirectory()
 	FString SpotProjectInfoArgs = TEXT("alpha service project-info --json");
 	FString SpotProjectInfoResult;
 	FString StdErr;
+	FString SpatialServiceStatusArgs = FString::Printf(TEXT("service status %s"), *GetDomainEnvironmentStr(bIsInChina));
```

Comment:
```suggestion
```

Reply:
Just spoke to @joshuahuburn about this. Spot is talking directly to spatiald so it doesn't need the domain arg because it's already been passed when starting the service.